src/Io/IniUtil.php

```diff
@@ -10,13 +10,13 @@
     /**
      * Convert a ini like size to a numeric size in bytes.
      *
-     * @param string $iniSetting
+     * @param string $size
      * @return int
```

Comment:
the point of this change is to assure @return int remains true

Reply:
See below, maybe use `number` type hint here? http://php.net/manual/de/language.pseudo-types.php#language.types.number